pkg/runner-provisioner/provisioner.go

```diff
@@ -104,28 +93,6 @@ func (p *RunnerProvisioner) getRealVMIP(vmIP string) (string, error) {
 	return p.envData.OrkaNodeIPMapping[vmIP], nil
 }
 
-func (p *RunnerProvisioner) DeprovisionRunner(ctx context.Context, runnerName string) {
```

Comment:
Discussed the removal of the deprovisioning logic w/ @ispasov - we will think of a way to improve this. For now it's an edge case if the job is cancelled before the runner is created - it will be left running but will pick up the next available job, and be deprovisioned when that job has completed